index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body class="container">
+    <header class="header">
+        <p class="name">Fernando Herrera</p>
+        <div>
+            <div class="hamb hamb1"></div>
+            <div class="hamb hamb2"></div>
+            <div class="hamb hamb3"></div>            
+        </div>     
```

Comment:
- [ ] You've done well to include the hamburger section for the toolbar. however as per the requirements please ensure that this is a `nav` tag and not just a `div`. Using a div here is more semantically correct as the hamburger icon is a section of the page that deals with navigation. 👍 
something like this would be great please:
 ```suggestion
        <nav>
            <div class="hamb hamb1"></div>
            <div class="hamb hamb2"></div>
            <div class="hamb hamb3"></div>            
        </nav>     
```